components/camel-debezium/src/main/java/org/apache/camel/component/debezium/DebeziumConnectorTypes.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.component.debezium;
+
+import io.debezium.connector.mysql.MySqlConnector;
+
+public enum DebeziumConnectorTypes {
+    // for the initial version of the component, we support MySQL
+    MYSQL("mysql");
```

Comment:
Are you planning the others, too? Note I don't know that much about Camel, so out of curiosity: why is it that you need to "know" about specific Debezium connectors at the Camel end of things?

Reply:
Good question, this is needed for the information when the users use the route URI, so if they users use this route ` from("debezium:mysql?[options]")` or posgtress .. etc. this will tell the component which connector to use and therefore parse the URI with the correct configurations that is suitable for the component. You can take a look at camel-kafka for example https://camel.apache.org/components/latest/kafka-component.html